scipy/stats/kde.py

```diff
@@ -170,14 +171,19 @@ def __init__(self, dataset, bw_method=None):
         self.d, self.n = self.dataset.shape
         self.set_bandwidth(bw_method=bw_method)
 
-    def evaluate(self, points):
-        """Evaluate the estimated pdf on a set of points.
+    def evaluate(self, points, lb=None, ub=None):
+        """Evaluate the estimated pdf on a set of points using a Gaussian
+        Kernel, boundary effects corrected using the reflection method.
```

Comment:
This will break in the layout of the web docs, where the first line is pulled into the listings, see e.g. http://docs.scipy.org/doc/scipy-0.17.0/reference/generated/scipy.stats.gaussian_kde.html#scipy.stats.gaussian_kde.
 The docstring format should be: the summary line, blank line, the rest of the docstring.  Ideally, keep the lines short enough so that the doc builder does not clip them.